Update scripts 1 and 2 to bdk-jvm:1.2.0

ItoroD opened the pull request:
I have updated the version to our latest 1.2.0 and made small import updates. I will add the rest of the scripts copying from your previous repo and maybe add some additional ones. I will make the version update in those as well. Once this is complete, I can mirror this for java repo

ItoroD committed:
chore: upgrade bdk-jvm to 1.2.0

ItoroD committed:
feat: Add old scripts back to this repo

ItoroD commented:
I figured I should add all old scripts back to this repo, so that we can clearly see the changes that are being made to the old scripts. Only script 1 and 2 have been updated to 1.2.0

thunderbiscuit reviewed (approved):
ACK 72c57beb1575820a80de208f30b946a519d26cb3. Thanks!

thunderbiscuit commented:
Actually I ACK'ed a bit too fast. Script #3 has updated the dependency but not the code, so it fails when you attempt to run it!

ItoroD commented:
Ahh yes I did not mean to update script 3. only 1 and 2 for now. I will revert that change. I plan to do those in separate PRs

ItoroD commented:
I have updated script 3. I have updated version and code for 1.2.0 on my machine. I think that slipped through for script 3. I have confirmed that all the rest are on previous versions. Will you rather I update all  versions in this PR. Script 3 might have to be removed, as I do not see that we have a `BlockchainConfig` as one of the classes in the bdk-jvm lib anymore
